coalib/parsing/Globbing.py

```diff
@@ -299,33 +306,35 @@ def relative_wildcard_glob(dirname, pattern):
     return result
 
 
-def relative_flat_glob(dirname, basename):
+def relative_flat_glob(dirname, basename, ignored_globs=None):
```

Comment:
`if basename == '**':
    relative_glob_function = relative_recursive_glob
elif has_wildcard(basename):
    relative_glob_function = relative_wildcard_glob
else:
    relative_glob_function = relative_flat_glob
 
if not dirname:
    for file in relative_glob_function(dirname, basename, ignored_globs):
        yield fille`

This is how the function looks like. And in case the if exists on 3rd path, the `relative_glob_function` is equal to `relative_flat_glob`. After this, the `relative_glob_function` is called with 2 parameters. So `relative_flat_glob` gets called with 2 parameters. If it expects only one, then it starts complaining.